cmd/containerd/command/main.go

```diff
@@ -260,9 +244,6 @@ can be used and modified as necessary as a custom configuration.`
 func serve(ctx gocontext.Context, l net.Listener, serveFunc func(net.Listener) error) {
 	path := l.Addr().String()
 	log.G(ctx).WithField("address", path).Info("serving...")
-	serveSpan, ctx := tracing.StartSpan(ctx, l.Addr().String())
-	defer tracing.StopSpan(serveSpan)
```

Comment:
This did effectively nothing since we return immediately after starting the grouting below.